cloud/pkg/cloudhub/config/config.go

```diff
@@ -0,0 +1,138 @@
+package config
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"sync"
+
+	"k8s.io/klog"
+
+	"github.com/kubeedge/beehive/pkg/common/config"
+)
+
+var c Configure
+var once sync.Once
+
+type Configure struct {
+	ProtocolWebsocket  bool
+	ProtocolQuic       bool
+	ProtocolUDS        bool
+	MaxIncomingStreams int
+	Address            string
+	Port               int
+	QuicPort           int
+	UDSAddress         string
+	KeepaliveInterval  int
+	Ca                 []byte
+	Cert               []byte
+	Key                []byte
+	WriteTimeout       int
+	NodeLimit          int
+}
+
+func InitConfigure() {
+	once.Do(func() {
```

Comment:
Do we need the once here?

Reply:
yes, Prevent global variable duplicate assignment.